gitevanhsu/book-reviews
# Feature/search

**Description**
1. User can search book with title.
2. Click book cover can go to review page.

**Test plan**
1. Search book with title.
2. Click book cover go to review page.
3. Check firestore save book information.
